src/class-fields/eval-err-contains-arguments.case

```diff
@@ -0,0 +1,29 @@
+// Copyright (C) 2017 Valerie Young. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+desc: `arguments` in StatementList of eval
```

Comment:
the initial ` triggers a failure on the yaml parser. This applies to the other cases as well.